docs/kratos/social-signin/30_apple.mdx

```diff
@@ -4,8 +4,60 @@ title: Add Apple as a social sign-in provider in Ory
 sidebar_label: Apple
 ---
 
+import JsonnetWarning from "../../_common/jsonnetwarning.mdx"
+
 # Apple
 
+````mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+<TabItem value="console" label="Ory Console" default>
+
+Follow these steps to add Apple as a social sign-in provider to your project using the Ory Console:
+
+1. Sign in to Ory Console and select [**Social Sign-in**](https://console.ory.sh/projects/current/social-signin).
+2. Click the switch next to the Apple logo to start the configuration.
+3. Copy the **Redirect URI** and save it for later use.
+4. Follow the [Sign in with Apple](https://developer.apple.com/documentation/sign_in_with_apple) guide to
+   [create an app, a service, and a private key using an Apple Developer account](https://developer.apple.com/documentation/sign_in_with_apple/implementing_user_authentication_with_sign_in_with_apple).
+5. Copy the **Services ID** from the Apple registered app to the **Client ID** field in the form on the Ory Console.
+5. Add the saved **Redirect URI** from Ory to the **Return URLs** of the Apple registered application.
+6. In the **Scopes** field of the form on the Ory Console, add the following scope:
+    - `email`
+7. Copy the following details from your registered application in Apple to the Ory Console form:
+    1. Copy the **Apple Team Id** to the corresponding field in the form on the Ory Console.
+    2. Copy the **Apple Private Key Id** to the corresponding field in the form on the Ory Console.
+    3. Copy the **Apple Private Key** to the corresponding field in the form on the Ory Console.
```

Comment:
Why `Id` and not `ID`? 
This nested list isn't the most elegant way to do this - you've basically have 3 sentences of the same structure where only one thing changes. It'd be better if you made this a single sentence.

Reply:
"Id" and not "ID", because the names of those fields are taken verbatim from Apple's UI (in other words, it's Apple's mistake -- otherwise, I would always use "ID").

Agreed that this list is not so elegant. I'll change it to improve the structure.

Reply:
Restructured this content (and parallel sections) in such a way that it just shows the field names in the list.